core/adapters/eth_tx_test.go

```diff
@@ -164,7 +164,7 @@ func TestEthTxAdapter_Perform_BPTXM(t *testing.T) {
 		jobRunID := models.NewID()
 		taskRunID := cltest.MustInsertTaskRun(t, store)
 		etx := cltest.MustInsertUnconfirmedEthTxWithBroadcastAttempt(t, store, 0, fromAddress)
-		store.DB.Exec(`INSERT INTO eth_task_run_txes (task_run_id, eth_tx_id) VALUES ($1, $2)`, taskRunID.UUID(), etx.ID)
+		store.DB.Exec(`INSERT INTO eth_task_run_txes (task_run_id, eth_tx_id) VALUES (?, ?)`, taskRunID.UUID(), etx.ID)
```

Comment:
I believe you can drop down to the raw SQL driver and keep the params, no?

`store.DB.DB().Exec()`

Also, lets handle the error here